.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2025-05-19 - Clear Button Pattern for Inputs
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)